### PR TITLE
[AMS-FT-102020] Ceyda Ulubas

### PR DESCRIPTION
**Pull request by ceydaulubas**
I've created HTML and css structures for NPM Clone.  If there are some mistakes on my codes, please correct me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,91 @@
...
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
```
Comment (LloydC) [inline index.html:3]: Your `<header>` should be nested inside the `<body>...</body>`. There should never be any HTML element that's between the `<head>...</head>` and `<body>...</body>`

File: index.html [hunk 2]
```diff
@@ -1,9 +1,91 @@
-NPM Clone Nifty Penguin Magic npm Enterprise Products Solutions Resources Docs Support Search Join Log In Build amazing
-things Essential JavaScript development tools that help you go to market faster and build powerful applications using
-modern open source code. See plans Join for free Bring the best of open source to your company npm is the tool used by
-over 11,000,000 JavaScript developers around the world. Your developers already use it. Your company depends on it.
-Create an Org and get more out of the tools your team already knows and loves. Zero configuration Create an org, add
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
```
Comment (LloydC) [inline index.html:9]: You should also have a `<!DOCTYPE html>` here.

File: index.html (hunk 1 above)
Comment (ceydaulubas) [reply index.html:3]: I edited their location.

File: index.html (hunk 2 above)
Comment (ceydaulubas) [reply index.html:9]: I added <!DOCTYPE html>. Meanwhile, I changed <svg >tag totag because <svg> wasn't supported with <!DOCTYPE html>.

**Comment by ceydaulubas** [comment]
I did my changes. Could you please check them?  Thank you for your review.

**Pull request closed without merging**